cfeScraping/index.js

```diff
@@ -45,10 +46,75 @@ ScrapPage.prototype.clickButton = async function (id, time) {
     await this.page.click(id);
     console.log('Searching...');
     await this.page.waitForTimeout(time);
-    return console.log('Search succesfull');
+    return console.log('Search successful');
   } catch (err) {
     return console.error('Error: ', err);
   }
 };
 
+// eslint-disable-next-line space-before-function-paren
+ScrapPage.prototype.getDataTable = async function () {
+  try {
+    const data = await this.page.evaluate(
+      (selector = 'table.k-selectable tr') => {
+        const elementos = document.querySelectorAll(selector);
+        const inf = [];
+        for (const el of elementos) {
+          inf.push(el.innerText);
+        }
+        return inf;
+        // eslint-disable-next-line comma-dangle
+      }
+    );
+    return data;
+  } catch (err) {
+    return console.error('Error: ', err);
+  }
+};
+
+// eslint-disable-next-line space-before-function-paren
+ScrapPage.prototype.printTable = async function (data, separator) {
+  try {
+    const dataArray = [];
+    for (let i = 0; i < data.length; i++) {
+      dataArray[i] = data[i].split(separator);
+    }
+    dataArray.shift();
+    const dataObject = dataArray.map((item) => {
```

Comment:
There are inconsistence in this method, should be only for print the data, the format of the data should be in `.getDataTable`  & the fuction recived to the `.map` should be general, remember we need this lib. general to reused on other pages. Imagine now we want use the .getDatatable of [this table](https://pokemondb.net/type), the map callback should be custom (pased by param if is needed to format the data)